packages/math/src/ObservablePoint.ts

```diff
@@ -2,28 +2,35 @@ import type { IPointData } from './IPointData';
 import type { IPoint } from './IPoint';
 
 /**
- * The Point object represents a location in a two-dimensional coordinate system, where x represents
- * the horizontal axis and y represents the vertical axis.
+ * The ObservablePoint object represents a location in a two-dimensional coordinate system, where `x` represents
+ * the position on the horizontal axis and `y` represents the position on the vertical axis.
  *
- * An ObservablePoint is a point that triggers a callback when the point's position is changed.
+ * An `ObservablePoint` is a point that triggers a callback when the point's position is changed.
  *
  * @class
  * @memberof PIXI
  * @implements IPoint
  */
+
 export class ObservablePoint<T = any> implements IPoint
 {
+    /** The callback function triggered when `x` and/or `y` are changed */
     public cb: (this: T) => any;
+
+    /** The owner of the callback */
     public scope: any;
+
     _x: number;
     _y: number;
 
     /**
-     * @param {Function} cb - callback when changed
-     * @param {object} scope - owner of callback
+     * Creates a new `ObservablePoint`
+     *
+     * @param cb - callback function triggered when `x` and/or `y` are changed
+     * @param scope - owner of callback
      * @param {number} [x=0] - position of the point on the x axis
```

Comment:
Explicit type / default was needed here. Not sure if there's a better way around this.